{
  "vulinfoid": "JVNDB-2019-010159",
  "vulinfodata": {
    "title": "Zcash における例外的な状態の処理に関する脆弱性",
    "vulinfodescription": {
      "overview": "Zcash には、例外的な状態の処理に関する脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "Zerocoin Electric Coin Company.LLC",
          "productname": "Zcash",
          "cpe": {
            "text": "cpe:/a:z.cash:zcash",
            "version": "2.2"
          },
          "versionnumber": [
            "2.0.7-3 未満"
          ]
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "5",
          "vector": "AV:N/AC:L/Au:N/C:P/I:N/A:N"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "5.3",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N"
        }
      ],
      "impactitem": {
        "description": "情報を取得される可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "Ignore exceptions when deserializing note plaintexts",
          "url": "https://github.com/zcash/zcash/commit/c1fbf8ab5d73cff5e1f45236995857c75ba4128d"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "v2.0.7-3",
          "url": "https://github.com/zcash/zcash/releases/tag/v2.0.7-3"
        },
        {
          "type": "vendor",
          "name": "Zcash",
          "vulinfoid": "Security Announcement 2019-09-24",
          "url": "https://z.cash/support/security/announcements/security-announcement-2019-09-24/"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2019-16930",
          "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-16930"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2019-16930",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-16930"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-755",
          "url": "https://cwe.mitre.org/data/definitions/755.html",
          "title": "例外的な状態における不適切な処理"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2019-10-07T17:50:23+09:00",
          "description": "[2019年10月07日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2019-10-07T17:50:23+09:00",
    "datelastupdated": "2019-10-07T17:50:23+09:00",
    "datepublic": "2019-09-24T00:00:00+09:00"
  }
}
